<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Vibhors-MacBook-Pro.local" ignored="0" tests="3" failures="0" timestamp="06 Feb 2020 10:27:10 GMT" time="0.037" errors="0">
  <testcase name="@BeforeTest initialization" time="0.012" classname="SignupTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at base.TestBase.getInstance(TestBase.java:37)
at base.TestBase.instantiateClasses(TestBase.java:29)
at base.TestBase.setup(TestBase.java:18)
at SignupTest.initialization(SignupTest.java:20)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.TestRunner.beforeRun(TestRunner.java:529)
at org.testng.TestRunner.run(TestRunner.java:497)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @BeforeTest initialization -->
  <testcase name="@AfterTest closeDriver" time="0.0" classname="SignupTest">
    <skipped/>
  </testcase> <!-- @AfterTest closeDriver -->
  <testcase name="eulaAcceptTest" time="0.0" classname="SignupTest">
    <skipped/>
  </testcase> <!-- eulaAcceptTest -->
  <testcase name="eulaDeclineTest" time="0.0" classname="SignupTest">
    <skipped/>
  </testcase> <!-- eulaDeclineTest -->
  <testcase name="logout" time="0.0" classname="SignupTest">
    <skipped/>
  </testcase> <!-- logout -->
</testsuite>
